debian/control

```diff
@@ -3,6 +3,11 @@ Section: admin
 Priority: extra
 Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
 Build-Depends: debhelper (>= 9),
+               libxml2-dev,
+               libxslt-dev,
+               python-dev,
+               lib32z1-dev,
```

Comment:
They are deps of lxml, without them lxml fails to install, which also causes travis to fail.